INSTALLING.md

```diff
@@ -27,6 +27,15 @@ sudo apt update
 sudo apt install software-properties-common=0.99.9.12 build-essential=12.8ubuntu1.1 python3.8-venv=3.8.10-0ubuntu1~20.04.9 libgoogle-glog-dev=0.4.0-1build1 libyaml-cpp-dev=0.6.2-4ubuntu1 libboost-all-dev=1.71.0.0ubuntu2 libsndfile1=1.0.28-7ubuntu0.2 libhwloc-dev graphviz
 ```
 
+Install Clang-17:
```

Comment:
Hmm should we combine these these two shell blocks into one, with a small instruction that says `Install system-level dependencies and Clang-17:`:

I'm concerned people are gonna skip over this block while reading mindlessly.

Reply:
Yea, might as well. Changing it rn..

Reply:
changed